README.md

```diff
@@ -437,31 +429,27 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
   end
   ~~~
 
-* Avoid hashes-as-optional-parameters in general. Does the method do too much?
```

Comment:
Did this move somewhere that I didn't see or do we want to get rid of it?

Reply:
I removed it because it's redundant with the rule below.